cmd/panacead/cmd/dep/hash_data.go

```diff
@@ -0,0 +1,41 @@
+package dep
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/cyberphone/json-canonicalization/go/src/webpki.org/jsoncanonicalizer"
+	"github.com/spf13/cobra"
+)
+
+func HashDataCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "hash-data [data-file-path]",
+		Short: "Hash a data by sha256 algorithm",
+		Long: `
+This command can hash a data by sha256 algorithm. The data will be encoded to canonical JSON format and printed as a hash value.`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			origData, err := os.ReadFile(args[0])
+			if err != nil {
+				return err
+			}
+
+			jsonData, err := jsoncanonicalizer.Transform(origData)
+			if err != nil {
+				return err
+			}
+
+			jsonDataHashBz := sha256.Sum256(jsonData)
+			_, err = fmt.Fprintln(cmd.OutOrStdout(), hex.EncodeToString(jsonDataHashBz[:]))
+			if err != nil {
+				return err
+			}
+			return nil
```

Comment:
Is there a reason you wrote this in `io.Writer` instead of just doing `fmt.Println(hex.EncodeToString(jsonDataHashBz[:]))`?

Reply:
Because I want to use `StdOut` as other commands(`encrypt-data`, `decrypt-data`) do. So I opened a #561 last time, too.

Reply:
In fact, the `fmt.Println()` writes to stdout. So, what you implemented is the same as `fmt.Println()` as @gyuguen said, unless you specify any output file parameter when you run the command (implemented by cobra).

For now, it's okay to use either of them.